argparse/templates/cwltool.j2

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env cwl-runner
+# This tool description was formed automatically by argparse2cwl ver. {{ version }}
+# To form again: $ {{ formcommand }} --generate_cwl_tool
```

Comment:
replace 'form' with 'generate'
This should also dynamically include any additional argparse2cwl options that were called such as `--basecommand`,  `--output_section`, or `--generate_outputs`